src/clarity-angular/data/datagrid/datagrid-row-detail.spec.ts

```diff
@@ -95,6 +96,30 @@ export default function(): void {
             expect(context.clarityElement.querySelectorAll(".datagrid-fixed-column").length).toBe(0);
         });
     });
+
+    fdescribe("DatagridRowDetail hide/show cell behavior", function() {
```

Comment:
yep, forgot to put it back after I made the first round of changes. 
Thanks for catching it.